public/app/features/plugins/pluginSettings.ts

```diff
@@ -1,19 +1,19 @@
 import { PluginMeta } from '@grafana/data';
-import { getBackendSrv } from '@grafana/runtime';
+import { BackendSrvRequest, getBackendSrv } from '@grafana/runtime';
 
 type PluginCache = {
   [key: string]: PluginMeta;
 };
 
 const pluginInfoCache: PluginCache = {};
 
-export function getPluginSettings(pluginId: string): Promise<PluginMeta> {
+export function getPluginSettings(pluginId: string, options?: Partial<BackendSrvRequest>): Promise<PluginMeta> {
```

Comment:
We're passing options here so we can disable the `showErrorAlert` property – if we don't then the user will see a error popup if a particular plugin is not installed.